compiler/damlc/tests/daml-test-files/List.daml

```diff
@@ -33,7 +29,7 @@ testSort6 = scenario do
 -- Utility called on by 'testSort4' and 'testSort5'
 check45 n m = do
     assert $ n == [1, 1, 1, 2, 3]
-    let o = map (flip snd ()) m
+    let o = map (`snd` ()) m
```

Comment:
`flip snd` is completely unreadable and infix notation does not make it better. Just use a lambda.
```suggestion
    let o = map (\(_, f) -> f ()) m
```

Reply:
As you wish